src/tlo/methods/cardio_metabolic_disorders.py

```diff
@@ -213,7 +213,7 @@ class CardioMetabolicDisorders(Module, GenericFirstAppointmentsMixin):
                   'nc_weight_loss_worked': Property(Types.BOOL,
                                                     'whether or not weight loss treatment worked'),
                   'nc_risk_score': Property(Types.INT, 'score to represent number of risk conditions the person has'),
-                  'ckd_total_dialysis_sessions': Property(Types.INT,
+                  'nc_ckd_total_dialysis_sessions': Property(Types.INT,
```

Comment:
I think the tests will have failed as the updates would also need to be made where this property is addressed.